src/components/TodoItem.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Todo } from '../types/Todo';
+import classNames from 'classnames';
+
+interface TodoItemProps {
+  todo: Todo;
+  handleDeleteTodo: (id: number) => void;
+  isDeleting: boolean;
+  loading?: boolean;
+}
+
+export const TodoItem: React.FC<TodoItemProps> = ({
+  todo,
+  handleDeleteTodo,
+  isDeleting,
+  loading,
+}) => {
+  return (
+    <div>
+      <div
+        key={todo.id}
+        data-cy="Todo"
+        className={`todo ${todo.completed ? 'completed' : ''}`}
+      >
+        <label className="todo__status-label" aria-label="я не знаю что тут">
```

Comment:
Use google for search this warning
```suggestion
        <label className="todo__status-label" aria-label="">
```